components/contract-pages/table/expansions/AirdropSection.tsx

```diff
@@ -41,30 +45,35 @@ export const AirdropSection: React.FC<IAirdropSection> = ({
     "Error transferring",
   );
 
-  const onSubmit = useCallback(
-    (data) => {
-      airdrop.mutate(
-        {
-          tokenId,
-          addresses: data.addresses,
-        },
-        {
-          onError,
-          onSuccess: () => {
-            onSuccess();
-            closeAllRows();
-          },
-        },
-      );
-    },
-    [airdrop, tokenId, onError, onSuccess, closeAllRows],
-  );
-
   const addresses = watch("addresses");
 
   return (
     <Stack pt={3}>
-      <form onSubmit={handleSubmit(onSubmit)}>
+      <form
+        onSubmit={handleSubmit((data) => {
+          airdrop.mutate(
+            {
+              tokenId,
+              addresses: data.addresses,
+            },
+            {
+              onError,
+              onSuccess: () => {
+                onSuccess();
+                trackEvent({
+                  category: "airdrop",
+                  action: "success",
+                  label: "edition",
+                  contractAddress: contract?.getAddress(),
+                  tokenId,
+                  path: asPath,
```

Comment:
couple of things here:
- `category` should be the most top level part of the structure, you will end up looking this up by looking at `category.action.label`
- `action` should be the user triggered type of event, so in this case `"airdrop"` would make sense here
- `label` should be a thing that you can then filter by (so in this case `success` would make sense here)

- contractAddress and tokenId make sense, path is not useful because every event will already include the full url etc

```suggestion
                  category: "edition",
                  action: "airdrop",
                  label: "success",
                  contractAddress: contract?.getAddress(),
                  tokenId
```